src/main/java/net/sf/jabref/logic/formatter/Formatters.java

```diff
@@ -45,7 +45,7 @@
 public class Formatters {
 
     public static final List<Formatter> CONVERTERS = Arrays.asList(
-            new HtmlToLatexFormatter(),
+            new HtmlToLatexFormatter(false), // TODO: Get the correct value from USE_CONVERT_TO_EQUATION
```

Comment:
The original reason here was actually to avoid an NPE rather than avoiding `Globals`. (That comment went missing it seems like.)